build.sbt

```diff
@@ -292,6 +292,8 @@ val mtagsSettings = List(
     baseDirectory.in(ThisBuild).value / "mtags",
     scalaVersion.value
   ),
+  // @note needed to deal with issues in https://github.com/scalameta/metals/pull/2157
+  sources in (Compile, doc) := Seq.empty,
```

Comment:
Since Metals is not a library, I imagine this will not be needed.

Reply:
Ahh, good point. I should have thought of this before. 👍